plugins/org.python.pydev.ast/src/org/python/pydev/ast/interpreter_managers/InterpreterInfo.java

```diff
@@ -1668,14 +1674,15 @@ public void setEnvVariables(String[] env) {
                     Log.log(e);
                 }
             } else {
-                Log.logInfo("Expected: " + new File(condaPrefix, "/etc/conda/activate.d")
+                Log.logInfo("Expected: " + new File(condaPrefix.getPath())
                         + " to exist to activate conda env.");
             }
         }
         if (hasEnvVarsToUpdate) {
             fillMapWithEnv(envVariables, computedMap, keysThatShouldNotBeUpdated, getStringVariableManager()); //will override the keys already there unless they're in keysThatShouldNotBeUpdated
         }
 
+        this.condaEnv = new HashMap<>(computedMap);
```

Comment:
This shouldn't be here... you still have to compute the `computedMap`, you only want to create a map with the variables inside of `this.activateCondaEnv` (and then update `computedMap` with the variables of the `condaEnv`).